src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -2625,6 +2641,13 @@ subroutine atm_divergence_damping_3d( state, diag, mesh, configs, dts, edgeStart
             end do
          end if ! edges for block-owned cells
       end do ! end loop over edges
+      !$acc end parallel
+
+      MPAS_ACC_TIMER_START('atm_divergence_damping_3d [ACC_data_xfer]')
+      !$acc exit data copyout(ru_p) delete(rtheta_pp, rtheta_pp_old, theta_m)
```

Comment:
For the sake of clarity, I would suggest either having separate directives for `copyout` and `delete`, or at least splitting the `copyout` and `delete` clauses onto separate lines.

Reply:
done